index.js

```diff
@@ -65,7 +55,7 @@ Converter.prototype.toBase64 = function () {
 
 Converter.prototype.toComment = function (options) {
   var base64 = this.toBase64();
-  var data = 'sourceMappingURL=data:application/json;base64,' + base64;
+  var data = 'sourceMappingURL=data:application/json;charset=utf-8;base64,' + base64;
```

Comment:
I'm a little bit concerned about this change.
It's not directly related to the goal of your PR.

Could you please link me to resources that show this works in all browsers?